Integrations/EDL/EDL.py

```diff
@@ -23,6 +25,11 @@ def write(msg):
 EDL_LIMIT_ERR_MSG: str = 'Please provide a valid integer for EDL Size'
 EDL_MISSING_REFRESH_ERR_MSG: str = 'Refresh Rate must be "number date_range_unit", examples: (2 hours, 4 minutes, ' \
                                    '6 months, 1 day, etc.)'
+''' REFORMATTING REGEXES '''
+# regexes from https://github.com/PaloAltoNetworks/minemeld-core/blob/master/minemeld/flask/feedredis.py#L36
```

Comment:
why do we need this comment?

Reply:
The regexes are from minemeld repo. I figured attribution was required.

Reply:
lets remove